tests/unittests/test_process.py

```diff
@@ -131,11 +131,11 @@ def test_filterDict_invalidDict_emptyReturn(self):
         """
         expected_status, expected_ret = False, None
         status, ret = hubblestack.extmods.fdg.process.filter_dict(
-            starting_dict=[1, 2, 3], chained={1: 'a', 2: 'b'})
+            starting_dict=[1, 2, 3], chained={1: 'a', 2: 'b'}, chained_status=True)
```

Comment:
You can also probably remove all of these `chained_status=True` with the changes I requested above.